### PR TITLE
Revisi 20230512

### PR DESCRIPTION
**Pull request by TempatKitaSoftware**
Perubahan :

- Perubahan Tampilan Progress Bar
- Template Form Surat Pengajuan
- Ubah Label Input dari "Profit" Menjadi "Margin Profit" di Form Pengajuan Additional
- Tambah Fitur Notifikasi
- Tambah Laporan Report Performa (Laporan Poin Pengajuan)
- Tambah Include Laba Rugi Kotor di Master Data Penyesuaian

### CODE REVIEW

File: system/proses/automailer.php [hunk 1]
```diff
@@ -0,0 +1,176 @@
+<?php
+
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\SMTP;
+use PHPMailer\PHPMailer\Exception;
+
+// Memanggil Fungsi Exception  di folder PHP Mailer
+require_once $_SERVER['DOCUMENT_ROOT'] . '/balistars2/assets/vendor/PHPMailer/src/Exception.php';
```
Comment (eriklukiman) [inline system/proses/automailer.php:8]: don't use hardcode in here. use $BASE_URL_HTML instead

File: system/proses/automailer.php [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+// Memanggil File Utama PHPMailer di folder PHP Mailer
+require_once $_SERVER['DOCUMENT_ROOT'] . '/balistars2/assets/vendor/PHPMailer/src/PHPMailer.php';
```
Comment (eriklukiman) [inline system/proses/automailer.php:11]: same here

File: system/proses/automailer.php [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+// Memanggil Fungsi SMTP di folder PHP Mailer
+require_once $_SERVER['DOCUMENT_ROOT'] . '/balistars2/assets/vendor/PHPMailer/src/SMTP.php';
```
Comment (eriklukiman) [inline system/proses/automailer.php:14]: same here

**Review by eriklukiman — CHANGES_REQUESTED** [review]

File: system/proses/automailer.php (hunk 1 above)
Comment (TempatKitaSoftware) [reply system/proses/automailer.php:8]: Okay, i will fix it

File: system/proses/automailer.php (hunk 2 above)
Comment (TempatKitaSoftware) [reply system/proses/automailer.php:11]: Okay, i will fix it

File: system/proses/automailer.php (hunk 3 above)
Comment (TempatKitaSoftware) [reply system/proses/automailer.php:14]: Okay, i will fix it

File: system/laporan/laporan_poin_pengajuan/.DS_Store
[the remark this answers is not in the document]
Comment (TempatKitaSoftware) [reply system/laporan/laporan_poin_pengajuan/.DS_Store:1]: Okay, i will remove it

**Pull request merged**